tests/simple/testdata/comparisons.textproto

```diff
@@ -1,2 +1,131 @@
 name: "comparisons"
 description: "Tests for boolean-valued functions and operators."
+section {
+  name: "Literal comparison"
+  description: "Comparing literals (no bindings)"
```

Comment:
Need to compare with `_!=_` too.  Either add those cases here, or make it clear that this section is only for `_==_` and add an empty section for `_!=_`.

Reply:
Agreed. Creating empty section to timebox this PR